test/symbol.cc

```diff
@@ -0,0 +1,51 @@
+#include <napi.h>
+
+using namespace Napi;
+
+Symbol CreateNewSymbolWithNoArgs(const Napi::CallbackInfo& info) {
+  (void)info;
```

Comment:
Why does it necessary do this?
```cpp
(void)info;
```

Reply:
I think this was to suppress compiler warnings for unused parameter, since the default Symbol constructor doesn't take in any values. But i can remove this function since it's not being called from js.

Reply:
To avoid compiler complains maybe you can try the following solution (comment the formal parameter of the function):
```cpp
Symbol CreateNewSymbolWithNoArgs(const Napi::CallbackInfo& /*info*/) {
```
Let me know if it worked.

Reply:
Yep it works. Thanks!